cmd/nebula-cert/keygen.go

```diff
@@ -59,8 +63,22 @@ func keygen(args []string, out io.Writer, errOut io.Writer) error {
 	} else {
 		switch *cf.curve {
 		case "25519", "X25519", "Curve25519", "CURVE25519":
-			pub, rawPriv = x25519Keypair()
-			curve = cert.Curve_CURVE25519
+			if *cf.inKeyPath == "" {
```

Comment:
Do you think this ought to be it's own "command"? Making this a bonus feature of `keygen` is potentially confusing.

Also, many users encrypt their private keys, we may need to support that. I think there's a helper function for this.... somewhere?